src/integrators/sdirk_integrators.jl

```diff
@@ -687,11 +678,14 @@ end
     end
   end
 
+  # TODO: Add extrapolant initial guess
```

Comment:
Ah OK, I just copied it from the not in-place method. So I'll remove it in both the in-place and not in-place method?